services/mouse.md

```diff
@@ -0,0 +1,42 @@
+# Mouse
+
+    identifier: 0x1885dc1c
+
+Control a device that acts as a mouse.
+
+## Commands
+
+    flags Button : u8 {
+        Right = 0x01
+        Middle = 0x04
+        Left = 0x02
+    }
+    enum ButtonEvent : u8 {
+        Up = 0x01
+        Down = 0x02
+        Click = 0x03
+        DoubleClick = 0x04
+    }
+    command set_button @ 0x80 {
+        buttons: Button
+        event: ButtonEvent
+    }
+
+Sets the up/down state of one or more buttons.
```

Comment:
We should specify:

A `Click` is the same as `Down` followed by `Up` after 100ms.
A `DoubleClick` is two clicks with 150ms gap between them (that is, 100ms first click, 150ms gap, 100ms second click).